docs/html/user_guide.rst

```diff
@@ -1478,45 +1486,38 @@ How to test
       - using ``pip install --force-reinstall`` to check whether
         it does what you think it should
       - using constraints files
+      - the "Setups to test with special attention" and "Examples to try" below
 
    -  If you have a build pipeline that depends on pip installing your
       dependencies for you, check that the new resolver does what you
       need.
 
-   -  If you'd like pip to default to using the new resolver, run ``pip
-      config set global.use-feature 2020-resolver`` (for more on that
-      and the alternate ``PIP_USE_FEATURE`` environment variable
-      option, see `issue 8661`_).
-
    -  Run your project’s CI (test suite, build process, etc.) using the
       new resolver, and let us know of any issues.
    -  If you have encountered resolver issues with pip in the past,
-      check whether the new resolver fixes them. Also, let us know if
-      the new resolver has issues with any workarounds you put in to
-      address the current resolver’s limitations. We’ll need to ensure
-      that people can transition off such workarounds smoothly.
+      check whether the new resolver fixes them, and read :ref:`Fixing
+      conflicting dependencies`. Also, let us know if the new resolver
+      has issues with any workarounds you put in to address the
+      current resolver’s limitations. We’ll need to ensure that people
```

Comment:
~~Is there a preferred way for people to get in touch? Would the [Dependency resolver failures / errors GH issue template ](https://github.com/pypa/pip/issues/new?labels%5B%5D=K%3A+UX&labels%5B%5D=K%3A+crash&labels%5B%5D=C%3A+new+resolver&labels%5B%5D=C%3A+dependency+resolution&template=resolver-failure.md) be the right way? If so, a link to it here would help if users have found this (maybe as a result of something breaking!).~~